config/application.rb

```diff
@@ -67,6 +67,16 @@ class Application < Rails::Application
     # Configure the default encoding used in templates for Ruby 1.9.
     config.encoding = "utf-8"
 
```

Comment:
comment why this is needed now? introduced  by new rails version?

Reply:
This comes from new rails version
https://stackoverflow.com/questions/72970170/upgrading-to-rails-6-1-6-1-causes-psychdisallowedclass-tried-to-load-unspecif
https://discuss.rubyonrails.org/t/cve-2022-32224-possible-rce-escalation-bug-with-serialized-columns-in-active-record/81017
@Robin481 told me how he solved it in his projects

Reply:
good, I added those links as comment just above this config block